pyroomacoustics/transform/stft.py

```diff
@@ -734,7 +767,7 @@ def synthesis(X, L, hop, win=None, zp_back=0, zp_front=0):
 
 
 def compute_synthesis_window(analysis_window, hop):
-    '''
+    """
```

Comment:
Is this the preferred standard ?

Reply:
I tried seeing if there was a standard online, main thing they suggest is consistency. Since most docstrings in this file are `"` I changed them to this.

Reply:
At some point we should rationalize the style... 😒 But I'm not sure when is a good time... I think this is good for now.

Reply:
I was about to make everything `'` (because it is less keys to type, is that your reasoning?) and then PyCharm suggested me to keep `"` for docstrings so I'm really not sure :neutral_face:

Reply:
I have been trying to consistently use ` '` (not sure why it is different in this file though). I have checked and [black](https://github.com/ambv/black) uses `"`, so we could use that. At some point, it would be could to just run everything through the formater.

Reply:
For now, let's just leave this file with `"` and progressively switch.